diskqueue.go

```diff
@@ -625,7 +644,16 @@ func (d *diskQueue) ioLoop() {
 			count = 0
 		}
 
-		if (d.readFileNum < d.writeFileNum) || (d.readPos < d.writePos) {
```

Comment:
this expression did not need to change (and the original is more concise)

Reply:
Yes, the change just for my debug, and forget change back.
I will change it back, and submit another pr.

Reply:
no need for another PR: you can do a git interactive-rebase, and force-push this branch, to make any change to this PR